apps/token-e2e/src/integration/view/home.cy.js

```diff
@@ -63,6 +63,39 @@ context('Home Page - verify elements on page', { tags: '@smoke' }, function () {
             .and('have.text', 'Browse, vote, and propose');
         });
       });
+      it('should show open or enacted proposals with proposal summary', function () {
+        cy.get('body').then(($body) => {
+          if (!$body.find('[data-testid="proposals-list-item"]').length) {
+            cy.createMarket();
```

Comment:
**praise (non-blocking)**: I haven't looked at these tests in ages. It's cool to see such a giant abstractin 😆